### DISCUSSION TITLE
Validating media items in the rules method

### DISCUSSION THREAD

**Discussion by bvtrfly** [spatie/laravel-data#800] [Q&A] [answered]
Using the Laravel Media Library package, I was wondering if there is a way to validate the media directly in the static rules method.

Right now it is not possible due to the method being static (so calling $this->... won't work). Is there a work around it or should I just stick with a normal form request?

**Comment by rubenvanassche** [comment, marked as answer]
I would just stick to the Form Request method, I had once a working thing but it was too hacky.

**Reply by bvtrfly** [reply]
Form request it is then. I appreciate your answer!
